benchmark/buffers/buffer-base64url-encode.js

```diff
@@ -0,0 +1,20 @@
+'use strict';
+const common = require('../common.js');
+
+const bench = common.createBenchmark(main, {
+  len: [64 * 1024 * 1024],
+  n: [32],
+}, {
+  test: { len: 256 },
+});
+
+function main({ n, len }) {
+  const b = Buffer.allocUnsafe(len);
+  let s = '';
+  let i;
+  for (i = 0; i < 256; ++i) s += String.fromCharCode(i);
+  for (i = 0; i < len; i += 256) b.write(s, i, 256, 'ascii');
+  bench.start();
+  for (i = 0; i < n; ++i) b.toString('base64url');
```

Comment:
Assign the result of the toString to a variable and then `assert.ok` to avoid v8 deadcode elimination